site/installing/standalone.md

```diff
@@ -3,4 +3,96 @@ title: Installing Weave Flux Standalone
 menu_order: 20
 ---
 
-_Content from deploy/standalone/README.md_
+Flux comes in several parts: 
+
+-   the command-line client **fluxctl**
+-   the service, which serves the API and
+-   the daemon, which carries out tasks on behalf of the service.
+
+Usually you would run the daemon yourself, and use the service that
+runs in Weave Cloud. But you can also run both of these in your own
+cluster.
+
+## Quick Install
+
+Install flux and its dependencies on a kubernetes cluster with:
+
+```
+kubectl create -f ./deploy/standalone/
+```
+
+This will create all the deployments and services that are contained 
+within the [standalone deploy directory](../../deploy/standalone/).
+
+### Install the fluxctl client
+
+Next download the latest version of the [fluxctl client from github](https://github.com/weaveworks/flux/releases/latest).
+
+For OSX, you can use this one-liner:
+
+```
+curl -L $(curl -s https://api.github.com/repos/weaveworks/flux/releases | grep browser_download_url | grep darwin | head -n 1 | cut -d '"' -f 4) -o /usr/local/bin/fluxctl && chmod +x fluxctl
```

Comment:
I mean, I don't think it's a good idea to parse JSON like this, using `jq` would be better, but it's an extra dependency and I'd prefer to avoid it. May be we should consider an approach that'd just fix the URL, e.g. `get.weave.works/fluxctl`, or provide packages. Another approach would be to depend on Docker, e.g. `docker run -v /usr/local/bin:/target weaveworks/fluxctl:installer`. I'd be much happier if we just pointed folks at the releases page for the time being, and document download command there, e.g. https://github.com/docker/compose/releases.